pjsip/src/pjsua-lib/pjsua_acc.c

```diff
@@ -2567,16 +2567,15 @@ static void regc_cb(struct pjsip_regc_cbparam *param)
 		if ((status != PJ_SUCCESS) &&
 		    pjsua_var.ua_cfg.cb.on_ip_change_progress)
 		{
-		    pjsua_ip_change_op_info ip_chg_info;
+			pjsua_ip_change_op_info ip_chg_info;
```

Comment:
revert these unnecessary white space changes, to avoid conflict with #3027 once merged.